pipelines/build_pipeline_pass.yaml

```diff
@@ -21,3 +30,7 @@ spec:
           value: "$(params.output-image)"
         - name: image-digest
           value: "$(params.image-digest)"
+    - name: clone-repository
+      taskRef:
+        name: clone-repository
+        bundle: quay.io/redhat-appstudio/example-tekton-bundle:clone-repository
```

Comment:
Why do we need this bundle? what if we don't include it here?

Reply:
This is the Tekton bundle that contains the task definition. In the future this will change to reference definitons in a git repo, but until then the best way to pacakge individual Tekton tasks and pipeline definitons is in bundle images.

Reply:
thanks!